src/test/java/teammates/test/cases/BaseTestCaseWithMinimalGaeEnvironment.java

```diff
@@ -14,6 +20,18 @@
 
     private LocalServiceTestHelper helper = new LocalServiceTestHelper();
 
+    /**
+     * Sets the environment of SystemProperty based on the URL.
+     */
+    @BeforeSuite
+    public void setUpEnvironment() throws MalformedURLException {
+        if (new URL(TestProperties.TEAMMATES_URL).getProtocol().equals("https")) {
```

Comment:
I still don't like the idea of using `https` to determine if the app is in production mode or not. Why not check if there is an `.appspot.com`?

Reply:
I agree with @wkurniawan07 mentioned which will also prevent the confusion in the environment value if someone sets the URL to say something like `http://teammates-john.appspot.com`.

Reply:
Makes more sense. Updated 👍